docs/tutorial/updates.md

```diff
@@ -10,6 +10,114 @@ The easiest and officially supported one is taking advantage of the built-in
 [Squirrel](https://github.com/Squirrel) framework and
 Electron's [autoUpdater](../api/auto-updater.md) module.
 
+## Using cloud object storage (serverless)
+
+For a simple serverless update flow, Electron's autoUpdater module can
+check if updates are available by pointing to a static storage URL
+containing latest release metadata.
+
+When a new release is available, this metadata needs to be published to
+cloud storage alongside the release itself. The metadata format is
+different for macOS and Windows.
+
+### Publishing release metadata
+
+With Electron Forge, you can set up static file storage updates by publishing
+metadata artifacts from the ZIP Maker (macOS) with `macUpdateManifestBaseUrl`
+and the Squirrel.Windows Maker (Windows) with `remoteReleases`.
+
+See Forge's [Auto updating from S3](https://www.electronforge.io/config/publishers/s3#auto-updating-from-s3)
+guide for an end-to-end example.
+
+<details>
+<summary>Manual publishing</summary>
+
+On macOS, Squirrel.Mac can receive updates by reading a `releases.json` file with the
+following JSON format:
+
+```json title='releases.json'
+{
+  "currentRelease": "1.2.3",
+  "releases": [
+    {
+      "version": "1.2.1",
+      "updateTo": {
+        "version": "1.2.1",
+        "pub_date": "2023-09-18T12:29:53+01:00",
+        "notes": "Theses are some release notes innit",
+        "name": "1.2.1",
+        "url": "https://mycompany.example.com/myapp/releases/myrelease"
+      }
+    },
+    {
+      "version": "1.2.3",
+      "updateTo": {
+        "version": "1.2.3",
+        "pub_date": "2024-09-18T12:29:53+01:00",
+        "notes": "Theses are some more release notes innit",
+        "name": "1.2.3",
+        "url": "https://mycompany.example.com/myapp/releases/myrelease3"
+      }
+    }
+  ]
+}
+```
+
+On Windows, Squirrel.Windows can receive updates by reading from the RELEASES
+file generated during the build process. This file details the `.nupkg` delta
+package to update to.
```

Comment:
You say delta here and then the example has `-full`? I think dropping `delta` leaves this readable and less confusing

Reply:
[eecd4d8](https://github.com/electron/electron/pull/42690/commits/eecd4d8177487ad486e0319020028e482944dc5d)